cpp/src/round/round.cu

```diff
@@ -209,6 +227,45 @@ std::unique_ptr<column> round_with(column_view const& input,
   return result;
 }
 
+template <typename T,
+          template <typename>
+          typename RoundFunctor,
+          typename std::enable_if_t<cudf::is_fixed_point<T>()>* = nullptr>
+std::unique_ptr<column> round_with(column_view const& input,
+                                   int32_t decimal_places,
+                                   cudaStream_t stream,
+                                   rmm::mr::device_memory_resource* mr)
+{
+  using namespace numeric;
+  using Type                   = device_storage_type_t<T>;
+  using FixedPointRoundFunctor = RoundFunctor<Type>;
+
+  // if rounding to more precision that fixed_point is capable of, just need to rescale
+  // note: decimal_places has the opposite sign of numeric::scale_type (therefore have to negate)
+  if (input.type().scale() > -decimal_places) {
```

Comment:
OMG another case where decimal_places is inverted. (just ranting)

Reply:
I think this is the same case :man_shrugging:

Reply:
Well, before it was inverted WRT mathematics (10^decimal_places is the opposite of what you would expect). Here it's inverted WRT the definition of fixed-point scale...